worker/src/test/java/gov/cms/ab2d/worker/processor/JobPreProcessorUnitTest.java

```diff
@@ -131,12 +132,30 @@ void whenJobIsInSubmittedStatus_ThenJobShouldBePutInProgress() throws Interrupte
         verify(jobRepository).save(Mockito.any());
     }
 
+    @DisplayName("Job is FAILED if contract is not attested")
+    @Test
```

Comment:
just a few line comments about mock tests also would be great for readability in future. Everything else looks great and Great job on finishing this super quickly!